src/Reference/Task7.java

```diff
@@ -0,0 +1,21 @@
+package Reference;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Task7 {
+
+    public static void main(String[] args) {
+
+        LocalDateTime now = LocalDateTime.now();
+        DateTimeFormatter divByDoublePoint = DateTimeFormatter.ofPattern("dd::MMM::yyyy");
+        DateTimeFormatter divByDoublePointWithHours = DateTimeFormatter.ofPattern("dd::MMM::yyyy HH::MM::SS");
+        DateTimeFormatter divBySlash = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+
+        System.out.println(
```

Comment:
Let's create a separate method for this. Something like:
printFormattedDate(LocalDateTime dateTime, String format).
We will have a single method and only 3 calls. And we will pass a date and a pattern to it.